### ISSUE TITLE
Can we have DLSS?

### ISSUE THREAD

**Issue by Niter88** [fholger/openvr_fsr#169]
I'd love to have support to DLSS on this, it runs a lot faster for RTX cards. The performance boost on VR would be insane.
If any of you have a fork with DLSS support, please, tell me. I would even help it with code and tests if you want to.

**Comment by sycspysycspy** [comment]
No, it's not what you think it is. DLSS has to be implemented in the game code to work probably, it not something can be done with a universal DLL injection. Correct me if I am wrong.

**Comment by Niter88** [comment]
The same as FSR, it has to be in the game but some bros inject it.
For SkyrimVR: https://www.nexusmods.com/skyrimspecialedition/mods/82179
the base plugin: https://www.nexusmods.com/skyrimspecialedition/mods/80343

This is just an example, but PureDark did a poor job so I avoid this plugin. Also if we have it here, it'll work in any game.

Only thing is, you can't distribute the download for DLSS, so the user would have to download it and drop it in the folder.

**Comment by sycspysycspy** [comment]
It's not the same as FSR, FSR is a post process and DLSS could never be implemented this way. The plugin is per game basis, and you can't expect this to work the same way.
